src/session.ts

```diff
@@ -82,22 +88,42 @@ export const get = async (sessionId: string): Promise<ISession> => {
 };
 
 export const commit = async (
+  req: express.Request,
   serviceId: string,
-  sessionId: string
+  sessionId: string,
+  options: IEveryAuthOptions
 ): Promise<{ identityId: string; tenantId: string; userId: string }> => {
   const profile = await getAuthedProfile();
   const baseUrl = getSessionUrl(profile);
 
   debug(`${sessionId}: committing`);
 
-  // Start the commit process
+  // Invoke callback if provided
+  if (options.onAuthorized) {
+    let result = await superagent
+      .get(`${baseUrl}/session/${sessionId}/`)
+      .set('User-Agent', EveryAuthVersion)
+      .set('Authorization', `Bearer ${profile.accessToken}`);
+    const ctx = {
+      serviceId,
+      tags: result.body.tags,
+    };
+    await options.onAuthorized(req, ctx);
+  }
```

Comment:
Should the session be committed with any updated `tags` after the onAuthorized call?

Reply:
Ideally, but I don't know how to make it per question on Slack, and it is not needed for the demos at the moment. We can enhance this API later.